Model/Push.php

```diff
@@ -487,11 +487,14 @@ private function receivePushCheckDuplicates($receivedStatusCode = null, $trxId =
             Giftcards::PAYMENT_METHOD_CODE,
             Transfer::PAYMENT_METHOD_CODE
         ];
+
+        $isRefund = isset($this->postData['brq_amount_credit']) && $this->postData['brq_amount_credit'] > 0;
+
         if ($payment
             && $payment->getMethod()
             && $receivedStatusCode
             && ($this->getTransactionType() == self::BUCK_PUSH_TYPE_TRANSACTION)
-            && (!in_array($payment->getMethod(), $ignoredPaymentMethods))
+            && (!in_array($payment->getMethod(), $ignoredPaymentMethods) || $isRefund)
```

Comment:
maybe also add brackets in the first check to isolate !, sth like this: ((!in_array($payment->getMethod(), $ignoredPaymentMethods)) || $isRefund)